core/src/main/java/nl/nn/adapterframework/jms/JmsRealmFactory.java

```diff
@@ -1,5 +1,5 @@
 /*
-   Copyright 2013, 2020 Nationale-Nederlanden
+   Copyright 2013-2020 Nationale-Nederlanden, 2021 WeAreFrank!
```

Comment:
`2013-2020` is not the same as `2013, 2020`.
The dash suggests that there are a 'lot' of changes in the whole period, whereas the comma suggests that there are some in 2013 and some in 2020